MainModule/Server/Core/Process.lua

```diff
@@ -1188,7 +1185,7 @@ return function(Vargs, GetEnv)
 					end
 				end--]=]
 
-				task.spawn(Functions.Donor, p)
+				task.defer(Functions.Donor, p)
```

Comment:
But, why?

Reply:
> But, why?

imo better to queue it up since its nothing important that you would want to call immediately, would lean more for performance i believe?

Reply:
In fact it it doesnt have to be queued it has a performance pentalty

Reply:
I always defaulted to task.defer and I've always hear it's just better practice lol